docs/src/pages.ts

```diff
@@ -285,6 +285,7 @@ const pages: MuiPage[] = [
       { pathname: '/discover-more/languages' },
     ],
   },
+  { pathname: '/performance/api', displayNav: false }, // TODO remove
```

Comment:
Used for testing the integration

Reply:
Can we remove this now?

Reply:
We could, it's meant to allow debugging the rendering https://deploy-preview-23726--material-ui.netlify.app/performance/api/.